homeassistant/components/zha/services.yaml

```diff
@@ -435,3 +435,36 @@ set_lock_user_code:
       example: 1234
       selector:
         text:
+
+change_channel:
```

Comment:
This is an admin task. We shouldn't expose that as a service. people shouldn't automate this.